src/PostgREST/Types.hs

```diff
@@ -115,6 +115,14 @@ data OrderTerm = OrderTerm {
 , otDirection :: Maybe OrderDirection
 , otNullOrder :: Maybe OrderNulls
 } deriving (Show, Eq)
+instance Monoid OrderTerm where
+  mempty = OrderTerm (mempty, mempty) Nothing Nothing
+  OrderTerm ("",Nothing) Nothing nn  `mappend` OrderTerm tm Nothing Nothing = OrderTerm tm Nothing nn
```

Comment:
All the cases in this `mappend` definition are confusing to me, and also I don't think they work right.

For instance, calling `/foo?order=a.asc.desc` blows up in a weird way:

```json
{
  "hint":null,
  "details":null,
  "code":"42601",
  "message":"zero-length delimited identifier at or near \"\"\"\""
}
```

What is the intended behavior? Perhaps something like

```haskell
mappend (OrderTerm at ad ano) (OrderTerm bt bd bno) =
  OrderTerm (at <> bt) (bd <|> ad) (bno <|> ano)
```
Meaning combine the terms, and choose whichever non-Nothing direction and order are last encountered.